extensions/wikia/WikiFactory/Hubs/WikiFactoryHub.php

```diff
@@ -224,7 +224,7 @@ public function getWikiCategories( $city_id, $active = 1 ) {
 
 		// query instead of lookup in AllCategories list
 		$categories = (new WikiaSQL())
-			->SELECT( "*" )
+			->SELECT( "cat_id", "cat_name", "cat_url", "cat_short", "cat_deprecated", "cat_active", "id", "city_id" )
```

Comment:
Please limit the number of columns we select. I found we use only `cat_id, cat_short`.
There is one wrong usage of this method in: https://github.com/Wikia/app/blob/76e4a4d1b5bdb352b75fd8e63b55ba3ae6c2752e/extensions/wikia/Track/Track.php#L86-L86 <-- is should be using `getWikiCategoryNames()`

Reply:
`hubFactory->getWikiVertical` is passed twice there - as `cd3` and `cd17`...